setup.py

```diff
@@ -1,43 +1,68 @@
 #!/usr/bin/env python
+import csv
+import imp
 import os
 import shutil
 import sys
 from setuptools import setup, find_packages
 
-VERSION = '0.0.1'
+DESCRIPTION = 'Wavelet scattering transforms in Python with GPU acceleration'
+VERSION = imp.load_source('kymatio.version', 'kymatio/version.py').version
```

Comment:
Interesting!

Questions:
1. I guess `imp` is part of the standard library?
2. Did you use some other `setup.py` as a template for this?

Reply:
Isn't `imp` deprecated in favor of `importlib`?

Reply:
Oh, I guess -- I actually only knew `importlib` ..

Reply:
yes I used the templates of librosa, scikit-learn, and pytorch, but mostly librosa
I will replace `imp` by `importlib`

Reply:
once this is done it should be good to merge

Reply:
fixed in 0aa74fb

Reply:
related: https://github.com/librosa/librosa/issues/788